### PR TITLE
About content updates cont.

### LINKED ISSUE #3896: Copy updates to PNI
(Old dev ticket for all links https://github.com/mozilla/foundation.mozilla.org/issues/3800)

Copy updates to PNI About pages.

### Add reference to *Privacy Included:

Copy updates are highlighted in yellow:

- [x] [How to Use This Guide](https://docs.google.com/document/d/12QuPQ_RXkmLlAlhCcPTX50-Nup2JskAIwC5ntcn8R4Q/edit): add a paragraph of copy to the bottom of the page

- [x] [Why We Made This Guide](https://docs.google.com/document/d/1rMo-TNMcHkgYOs2JSP8UxpW6AHdrx3N2odbL-uRP1Xs/edit): edit and add a new paragraph towards the bottom of the page

- [x] [Press](https://docs.google.com/document/d/153YTZ2edXYxMUtFAXyX34d7-WlKCW4xC9mYQd7i36JA/edit#heading=h.2ws9vo67f5s): add a sentence to the bottom of the page

### Add reference to Mozilla:

Copy updates are highlighted in blue:

- [x] Update first sentence and add context about Mozilla at the bottom of "Why We Made This Guide"

### Update order of nav items:

- [x] Switch the order of page so "Why we made this guide" is the first subnav item

### PR DESCRIPTION
**Pull request by youriwims**
Closes #3896
Related PRs/issues #3800

### CODE REVIEW

**Comment by kristinashu** [comment]
I see now what you mean about the url. Agree, we shouldn't update them now. But could you make the main nav item `About the Guide` link to ...`/privacynotincluded/about/why` instead of ...`/privacynotincluded/about`? It would be better if users landed on the `why` first. But do you foresee any issues with that? 

Copy looks good to me but I've just asked Jen to double check.

**Comment by kristinashu** [comment]
From Jen, please don't bold the new text at the bottom of the Press page (sorry, she thinks it's **too** bold).

Rest of the text looks good!

**Comment by youriwims** [comment]
> I see now what you mean about the url. Agree, we shouldn't update them now. But could you make the main nav item `About the Guide` link to ...`/privacynotincluded/about/why` instead of ...`/privacynotincluded/about`? It would be better if users landed on the `why` first. But do you foresee any issues with that?
> 
> Copy looks good to me but I've just asked Jen to double check.

Hmm, no I don't really foresee any issue with it other than the small amount of users who have already clicked that navitem may be a bit 'thrown off' to find it going to another page, but _again_ all of the about subpages are really easy to navigate through...so, shouldn't really be an issue :)

**Comment by youriwims** [comment]
Ready for re-review @kristinashu.

File: network-api/networkapi/buyersguide/templates/about/how_to_use.html [hunk 1]
```diff
@@ -183,4 +183,12 @@ <h1 class="h2-heading">
     </p>
   </section>
 
+  <section aria-labelledby="further-reading">
+    <p id="further-reading"><b>Further reading</b></p>
```
Comment (Pomax) [inline network-api/networkapi/buyersguide/templates/about/how_to_use.html]: we probably don't want a bold tag here, but CSS styling?

**Pull request merged**